po/fr.po

```diff
@@ -0,0 +1,5489 @@
+# French translation for flatpak.
+# Copyright (C) 2021 Flatpak team and others.
+# This file is distributed under the same license as the flatpak package.
+# Hari Rana / TheEvilSkeleton <theevilskeleton@riseup.net>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: flatpak 1.11.3\n"
+"Report-Msgid-Bugs-To: https://github.com/flatpak/flatpak/issues\n"
+"POT-Creation-Date: 2021-11-08 20:15-0600\n"
+"PO-Revision-Date: 2021-11-09 08:11-0500\n"
+"Last-Translator: FULL NAME <theevilskeleton@riseup.net>\n"
```

Comment:
"FULL NAME" should probably be replaced by your actual name.

Reply:
Thanks. I would've sworn I put my name.